lib/Models/GeoJsonCatalogItem.ts

```diff
@@ -0,0 +1,674 @@
+                        outlineColor: getColor(
+                            defaultValue(
+                                properties.stroke,
+                                options.polygonStroke
+                            )
+                        )
+                    });
+                    if (isDefined(properties["marker-opacity"])) {
+                        // not part of SimpleStyle spec, but why not?
+                        const color: Color = getPropertyValue(
+                            entity.point.color
+                        );
+                        color.alpha = parseFloat(properties["marker-opacity"]);
+                    }
+
+                    entity.billboard = (undefined as unknown) as BillboardGraphics;
+                }
+                if (
+                    isDefined(entity.billboard) &&
+                    isDefined(properties["marker-opacity"])
+                ) {
+                    entity.billboard.color = new ConstantProperty(
+                        new Color(
+                            1.0,
+                            1.0,
+                            1.0,
+                            parseFloat(properties["marker-opacity"])
+                        )
+                    );
+                }
+
+                // Cesium on Windows can't render polygons with a stroke-width > 1.0.  And even on other platforms it
+                // looks bad because WebGL doesn't mitre the lines together nicely.
+                // As a workaround for the special case where the polygon is unfilled anyway, change it to a polyline.
+                if (
+                    isDefined(entity.polygon) &&
+                    polygonHasWideOutline(entity.polygon) &&
+                    !polygonIsFilled(entity.polygon)
+                ) {
+                    entity.polyline = new PolylineGraphics();
+                    entity.polyline.show = entity.polygon.show;
+
+                    if (isDefined(entity.polygon.outlineColor)) {
+                        entity.polyline.material = new ColorMaterialProperty(
+                            entity.polygon.outlineColor
+                        );
+                    }
+
+                    var hierarchy: PolygonHierarchy = getPropertyValue(
+                        entity.polygon.hierarchy
+                    );
+
+                    var positions = hierarchy.positions;
+                    closePolyline(positions);
+
+                    entity.polyline.positions = new ConstantProperty(positions);
+                    entity.polyline.width = getPropertyValue(
+                        entity.polygon.outlineWidth
+                    );
+
+                    createEntitiesFromHoles(entities, hierarchy.holes, entity);
+
+                    entity.polygon = (undefined as unknown) as PolygonGraphics;
+                }
+            }
+            return dataSource;
+        });
+    }
+}
+
+function nameIsDerivedFromUrl(name: string, url?: string) {
```

Comment:
I don't think this is used anymore, because the strata system eliminates the need for it 🎉.